code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -157,14 +157,12 @@
 	for(var/c in assembly_components)
 		var/obj/item/integrated_circuit/circuit = c
 		if(circuit.removable)
-			HTML += "<a href='?src=[REF(circuit)]'>[circuit.displayed_name]</a> | "
-			HTML += "<a href='?src=[REF(circuit)];rename=1;return=1'>\[Rename\]</a> | "
```

Comment:
Should this also be done for the built in circuits at 143?

Reply:
Yes, I look at that later